web/app/menu/menu.settings.tpl.html

```diff
@@ -32,20 +32,40 @@ <h3 translate-keep-content translate="dashboard.settings.dialog.title">Dashboard
                             <input name="name" type="text" ng-model="form.name" class="form-control" />
                         </div>
                     </div>
-                    <div class="form-group" ng-class="{error: _form.tile.background_image.$error && _form.submitted}">
-                        <label class="control-label col-lg-3 col-md-3" translate-keep-content translate="dashboard.settings.tile.background_image_url">Background Image URL</label>
+
+                    <div class="form-group" ng-class="{error: _form.tile.background_type.$error && _form.submitted}">
+                        <label class="control-label col-lg-3 col-md-3" translate-keep-content translate="dashboard.settings.tile.background_type">Background Type</label>
                         <div class="col-lg-9 col-md-9">
-                            <input name="name" type="text" ng-model="form.tile.background_image" class="form-control" />
+                            <select name="background_type" ng-model="form.tile.background_type" class="form-control">
+                                <option value="default" translate-keep-content translate="dashboard.settings.tile.background.type.default">Default</option>
+                                <option value="url" translate-keep-content translate="dashboard.settings.tile.background.type.url">URL</option>
+                                <option value="color" translate-keep-content translate="dashboard.settings.tile.background.type.color">Color</option>
```

Comment:
"Color" is actually one the "common terms" I devised and is already translated in main.pot so you can simply use 
the "translate" directive without any further indication :)